pkg/kube/client.go

```diff
@@ -398,7 +411,12 @@ func newClientInternal(clientFactory *clientFactory, revision string) (*client,
 	if err != nil {
 		return nil, err
 	}
-	c.kubeInformer = informers.NewSharedInformerFactory(c.kube, resyncInterval)
+
+	c.kubeInformer = kubeinformer.NewSharedInformerFactoryWithOptions(
+		c.kube,
+		resyncInterval,
+		kubeinformer.WithNamespaces(), // Maistra needs to start with an empty namespace set.
```

Comment:
This shouldnt be here anymore. I think you'll need to add another commit by Marko to this PR to align with changes in xns-informer. It's this one: https://github.com/maistra/istio/commit/26ac5592c9bbd983544db6fcab4b232ec16c480b

Reply:
Damn, I see that it is making changes to both ior and this :/